cloud2cloud-connector/swagger.yaml

```diff
@@ -51,141 +51,144 @@ paths:
       responses:
         307:
           description: redirect to get authorization code
       security:
-      - c2c_connector_auth:
-        - empty
+        - c2c_connector_auth:
+            - empty
   /api/v1/clouds/{cloudId}/accounts/{accountId}:
     delete:
       tags:
         - Accounts
       summary: Delete linked account
       parameters:
-        - $ref: '#/components/parameters/cloudId'
-        - $ref: '#/components/parameters/accountId'
+        - $ref: "#/components/parameters/cloudId"
+        - $ref: "#/components/parameters/accountId"
       responses:
         200:
           description: successful operation
       security:
-      - c2c_connector_auth:
-        - empty
+        - c2c_connector_auth:
+            - empty
   /api/v1/oauthCallback:
     get:
       tags:
         - Accounts
       summary: Used for get authorization code for linked account
       parameters:
-        - $ref: '#/components/parameters/oauthCallback'
+        - $ref: "#/components/parameters/oauthCallback"
       responses:
         200:
           description: redirect to get autorization code
   /api/v1/events:
-    $ref: 'https://raw.githubusercontent.com/openconnectivityfoundation/core-extensions/ocfcloud-openapi/swagger2.0/oic.r.cloudopenapi.swagger.json#/paths/%2F%7BeventsUrl%7D'
+    $ref: "https://raw.githubusercontent.com/openconnectivityfoundation/core-extensions/ocfcloud-openapi/swagger2.0/oic.r.cloudopenapi.swagger.json#/paths/%2F%7BeventsUrl%7D"
 components:
   schemas:
     LinkedClouds:
       type: object
       properties:
         cloud:
-          $ref: '#/components/schemas/LinkedCloud'
+          $ref: "#/components/schemas/LinkedCloud"
     LinkedCloud:
       type: object
       properties:
-        Id:
+        id:
           type: string
           format: uuid
-        Name:
+        name:
           type: string
-        OAuth:
+        oauth:
           type: object
           properties:
-            ClientId:
+            clientID:
               type: string
-            ClientSecret:
+            clientSecret:
               type: string
-            Scopes:
+            scopes:
               type: array
               items:
                 type: string
-            Endpoint:
-              type: object
-              properties:
-                AuthUrl:
-                  type: string
-                TokenUrl:
-                  type: string
-                AuthStyle:
-                  type: string
-                  enum: [ AutoDetect, InParams, InHeader ]
-            Audience:
+            authURL:
               type: string
-            RedirectUrl:
+            tokenURL:
               type: string
-            AccessType:
+            authStyle:
               type: string
-            ResponseType:
+              enum: [autoDetect, inParams, inHeader]
+            audience:
               type: string
-            ResponseMode:
+            redirectURL:
```

Comment:
@ondrejtomcik @jkralik: How do we deal with acronyms in field names? It's inconsistent currently.

URL vs Url
```
type SubscriptionRequest struct {
	URL           string      `json:"eventsUrl"`
...
```

vs. 

```
type HTTPConfig struct {
	EventsURL     string              `yaml:"eventsURL" json:"eventsURL"`
...
```

ID vs. Id:
```
type Config struct {
	ClientID         string    `yaml:"clientID" json:"clientID"`
...
```
vs

```
type PublicConfiguration struct {
...
	DeviceIDClaim            string        `yaml:"deviceIdClaim" json:"deviceIdClaim"`
	HubID                    string        `yaml:"hubId" json:"hubId"`
...
```

Reply:
YAML definition should from the property name point of view match the go struct property name. Only difference are ID -> Id, URL -> Url. So `SubscriptionRequest.URL` should be IMHO renamed to `eventsURL`, and keep json `eventsUrl`. `clientID` to `clientId`

Reply:
yeah, I will rename the golang struct field, must have been an oversight that it doesn't match. 

actually, I could easily modify my python yaml tags validator to check that tags match case insensitive name of the go field and complain if it doesn't.